### ISSUE TITLE
TypeError: n.replace is not a function 

### ISSUE THREAD

**Issue by Salamandar** [gadgetchnnel/lovelace-card-templater#31]
I had a gauge card working fine until this week. Now the gauge is not showing any value. 
Here is the card : 
```
type: 'custom:card-templater'
entities:
  - sensor.doors_total
  - sensor.doors_open
card:
  type: gauge
  entity: sensor.doors_open
  unit: porte(s) ouvertes
  min: -0.02
  max_template: '{{ states.sensor.doors_total.state }}'
  name_template: '{{ "sur " + states.sensor.doors_total.state + " portes au total" }}'
```

(the min is here to show a small blue line in the gauge even if the value is zero)

It looks like `max_template` does not work anymore. `name_template` works fine.

Oh, and this works :
```
  max_template: "1"
```

The system log shows this : 
```
<my domain>/hacsfiles/lovelace-card-templater/lovelace-card-templater.js:10:2737 TypeError: n.replace is not a function 
```

Any idea why this doesn't work anymore ? 
Thanks !

**Comment by Salamandar** [comment]
The browser debugger tells me that when the exception occurs, `n: 2`.
It looks like integers aren't supported anymore in templates ?

**Comment by gadgetchnnel** [comment]
Should be fixed in the latest version, 0.0.14. Can you please try updating and see if it fixes it for you?

**Comment by Salamandar** [comment]
Looks like it, thanks !!
I can't check if listening on the variable works (because it's the total count of my devices, and I don't have spares to test runtime changes), but the gauge is working properly now.

Closing !
